### DISCUSSION TITLE
We welcome contributions!

### DISCUSSION THREAD

**Discussion by jkcso** [skills/secure-code-game#37] [Announcements]
By following our [contribution guideline](https://github.com/skills/secure-code-game/blob/main/CONTRIBUTING.md), you can unleash your creative energy and submit new levels to our game! Your participation can have a significant impact, and we invite you to join us on this exciting journey.

**Comment by anshitmishra** [comment]
Hello @jkcso i am anshit looking for first contribution in this repo would you please suggest me any issue

**Reply by jkcso** [reply]
Nice to meet you @anshitmishra and it's so exciting to know that you wanna contribute! Please check out our [contribution guideline](https://github.com/skills/secure-code-game/blob/main/CONTRIBUTING.md) and let me know what you think you can work on!

**Reply by anshitmishra** [reply]
Thanks @jkcso i am looking forward for creating a proposal
